Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt-get install -y r-base r-base-dev
 RUN apt-get install -y libssl-dev libcurl4-openssl-dev libxml2-dev jq pip sudo python3-venv
 
 # cmake is needed to install some packages
-RUN apt-get install -y cmake
+RUN apt-get install -y cmake libxml2 libglpk-dev libicu-dev libicu66
```

Comment:
What is the justification for adding the additional dependencies here rather than, say, in the DCA build?  For example, do you know that these are dependencies of other Shiny app's?

This build already includes `libxml2-dev`.   Does`libxml2` have something that `libxml2-dev` does not?